scripts/o3de/o3de/repo_properties.py

```diff
@@ -17,14 +17,15 @@
 import logging
 import hashlib
 import shutil
+import sys
+import urllib.parse
 from packaging.version import Version, InvalidVersion
 from packaging.specifiers import SpecifierSet
-from o3de import manifest, utils, validation
+from o3de import manifest, utils, validation, github_utils
```

Comment:
You shouldn't need to import `github_utils`, the function you want is in `utils`
```suggestion
from o3de import manifest, utils, validation
```